autoware.repos

```diff
@@ -12,9 +12,17 @@ repositories:
     type: git
     url: https://github.com/autowarefoundation/autoware_internal_msgs.git
     version: main
-  core/autoware_common:
+  core/autoware_cmake:
     type: git
-    url: https://github.com/autowarefoundation/autoware_common.git
+    url: https://github.com/autowarefoundation/autoware_cmake.git
+    version: main
+  core/autoware_utils:
+    type: git
+    url: https://github.com/autowarefoundation/autoware_utils.git
+    version: main
+  core/autoware_lanelet2_extension:
+    type: git
+    url: https://github.com/autowarefoundation/autoware_lanelet2_extension.git
```

Comment:
Could you extract this part as a separate PR?
I would like to merge this PR in the following order so that there is not build break happening:

1. Merge the new PR which adds autoware_lanelet2_extension to repos file (It should be okay because the package name is different)
2. Merge  https://github.com/autowarefoundation/autoware.universe/pull/7640
3. Merge this PR which replaces autoware_common with autoware_cmake and autoware_utils (No change needed to Universe, right?)

Reply:
That is good idea so I made the 1st PR https://github.com/autowarefoundation/autoware/pull/4912.